be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -251,9 +251,9 @@ final case class DbActor(
         case failure              => sender() ! failure.recover(Status.Failure(_))
       }
 
-    case ReadElectionData(electionId) =>
+    case ReadElectionData(electionId, laoId) =>
       log.info(s"Actor $self (db) received a ReadElectionData request for election '$electionId'")
-      Try(readElectionData(electionId)) match {
+      Try(readElectionData(electionId, laoId)) match {
```

Comment:
I believe you forgot to switch laoId and electionId to match the function definition